launchable/utils/http_client.py

```diff
@@ -51,7 +64,14 @@ def request(self, method, sub_path, payload=None, timeout=(5, 60), compress=Fals
 
         headers = self._headers(compress)
 
-        Logger().audit(AUDIT_LOG_FORMAT.format(method, url, headers, payload))
+        Logger().audit(AUDIT_LOG_FORMAT.format(
+            "(DRY RUN) " if self.dry_run else "", method, url, headers, payload))
+
+        if self.dry_run and method.upper() not in ["HEAD", "GET"]:
+            return DryRunResponse(status_code=200, payload={
```

Comment:
There are several POST requests issued with this LaunchableClient. Don't we need to change the payload based on which endpoint it queries against?

Reply:
The following places are where the cli is looking at the contents of the response.

- json()['testPaths'] and json()['rest'] in `split_subset`.
- json()['id'] in `record session`.

So I store them all together here.